packages/app/lib/ui/widgets/refresh_provider.dart

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+import 'package:flutter/widgets.dart';
+
+enum RefreshKeys {
```

Comment:
Very nice pattern :smile:

Reply:
I hope thats irony 😝

Reply:
I mean, kind of, we needed a fix for it and this isn't a bad one :+1: